Dockerfile

```diff
@@ -15,7 +15,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends curl gnupg git
 RUN apt-get install -y software-properties-common && add-apt-repository -y ppa:ondrej/php
 RUN apt-get update && apt-get install -y --no-install-recommends php8.1 php8.1-curl php8.1-iconv php8.1-mbstring php8.1-bcmath php8.1-gmp
 # Node
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get update
+RUN apt-get install -y ca-certificates curl gnupg
+RUN mkdir -p /etc/apt/keyrings
+RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+ENV NODE_MAJOR=16
```

Comment:
@carlosmiei should we use nodejs LTS (20) version here?

Reply:
@sc0Vu yeah I think we can bump it

Reply:
gm

newest commit bumps nodejs version to 20. i tested by building container and building ccxt from within container. lgtm.